### PR TITLE
Add testing tool to 'What's new'.

### PR DESCRIPTION
**Pull request by jpmedley**
@oliverdunk, thank you for the text you sent me over chat. I thought we could use a bit more about the tool. I went to your readme and discovered I that I liked your readme more than I liked what you sent. (Don't be afraid to quote yourself.) 

I'm hereby proposing new text based on the readme. Please make sure I didn't get anything wrong.

P.S. I'm slating this for Monday so that side panel can have a bit more time at the top.

### CODE REVIEW

File: site/en/docs/extensions/whatsnew/index.md [hunk 1]
```diff
@@ -15,6 +15,12 @@ tags:
 
 Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.
 
+### New tool: New Side Panel API {: #side-panel-launch }
```
Comment (AmySteam) [inline site/en/docs/extensions/whatsnew/index.md:18]:
```suggestion
### New tool: Extension Update Testing Tool {: #update-testing-tool }
```

File: site/en/docs/extensions/whatsnew/index.md [hunk 2]
```diff
@@ -15,6 +15,12 @@ tags:
...
+
+<p class="color-secondary-text type--caption">Posted on <time>June 5, 2023</time></p>
+
+We've just released the [Extension Update Testing Tool](https://github.com/GoogleChromeLabs/extension-update-testing-tool), a local extension update server that can be used for testing updates to Chrome Extensions during local development, including permission grants. Additionally, the tool shows: the user's update flow, including keeping an extension disabled until a user grants permissions; a migration flow that's similar to what users see when updating form the Chrome Web Store. This tool is also useful for simulating the permission changes requested when updatng an extension from Manifest V2 to Manifest V3.
```
Comment (oliverdunk) [inline site/en/docs/extensions/whatsnew/index.md:22]: Made a few tweaks (the three different things we were calling out are all really part of the same flow, so I wanted to make that clearer):

```suggestion
We've just released the [Extension Update Testing Tool](https://github.com/GoogleChromeLabs/extension-update-testing-tool), a local extension update server that can be used for testing updates to Chrome Extensions during local development, including permission grants. The tool shows the user's update flow, including keeping an extension disabled until a user grants any newly requested permissions. This tool is particularly useful for simulating the permission changes requested when updating an extension from Manifest V2 to Manifest V3.
```

**Comment by jpmedley** [comment]
@AmySteam Can you give me a quick LGTM? Thanks

**Review by AmySteam — APPROVED** [review]
👍

**Pull request merged**